src/__tests__/renderer.test.tsx

```diff
@@ -13,9 +13,34 @@ import {
   Mention,
   ProgressBar,
   Message,
+  MessageText,
 } from '..';
 
 describe('slack jsx', () => {
+  it('message with complex fallback text', () => {
+    const message = (
+      <Message
+        text={
+          <MessageText>
```

Comment:
why not just use `PlainText` or `MarkdownText` (see `AnyText`) for the `text` component property instead of creating a new `MessageText` component?

oh... do the props not align the same as with the `*Text` components?

Reply:
yeah the main thing being the `type: mrkdwn` vs `mrkdwn: true`

Reply:
OH ok yeah... then in that case, i think just better naming of the component might clarify things...

in general i've been asking myself this same question about a lot of various payload types.. eg. section.accessory/fields, etc.